### PR TITLE
Set up environment for Parameter test

### PR DESCRIPTION
**Pull request by Jiajia-Cui**

### CODE REVIEW

File: vega_sim/scenario/curve_market_maker/scenario.py [hunk 1]
```diff
@@ -220,6 +222,18 @@ def set_up_background_market(
...
         )
+
+        info_trader = InformedTrader(
+            wallet_name = INFORMED_WALLET.name,
+            wallet_pass = INFORMED_WALLET.passphrase,
+            price_process = price_process,
+            market_name = market_name,
+            asset_name = asset_name,
+            initial_asset_mint = self.initial_asset_mint,
+            # proportion_taken = self.proportion_taken,
```
Comment (TomMcL) [inline vega_sim/scenario/curve_market_maker/scenario.py]: This should be passed in as one of the many many parameters to this scenario 😄

File: vega_sim/scenario/ideal_market_maker/agents.py [hunk 2]
```diff
@@ -37,6 +40,8 @@ def __init__(
         wallet_pass: str,
         terminate_wallet_name: str,
         terminate_wallet_pass: str,
+        # informed_wallet_name: str,
+        # informed_wallet_pass: str,
```
Comment (TomMcL) [inline vega_sim/scenario/ideal_market_maker/agents.py]: Don't think this is needed?

File: vega_sim/parameter_test/parameter/configs_parameter_test.py [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+)
+from vega_sim.parameter_test.parameter.experiment import (
+    FILE_PATTERN,
+    FILE_PATTERN_LOB,
+)
+from vega_sim.scenario.common.utils.price_process import (
+    Granularity,
+    get_historic_price_series,
+)
+
```
Comment (TomMcL) [inline vega_sim/parameter_test/parameter/configs_parameter_test.py]: What is this file for?

**Review by TomMcL — CHANGES_REQUESTED** [review]
Can we add them to the 'comprehensive' scenario too?

File: vega_sim/scenario/curve_market_maker/scenario.py (hunk 1 above)
Comment (Jiajia-Cui) [reply vega_sim/scenario/curve_market_maker/scenario.py]: how should I set it up? I used `self.proportion_taken `but its not happy about it

File: vega_sim/parameter_test/parameter/configs_parameter_test.py (hunk 3 above)
Comment (Jiajia-Cui) [reply vega_sim/parameter_test/parameter/configs_parameter_test.py]: I added it earlier, and then decided to not using it:) I will delete it.

**Comment by TomMcL** [comment]
Could you clear the notebooks' outputs please? 👍

File: vega_sim/scenario/ideal_market_maker_v2/agents.py [hunk 4]
```diff
@@ -44,6 +48,8 @@ def __init__(
         wallet_pass: str,
         terminate_wallet_name: str,
         terminate_wallet_pass: str,
+        informed_wallet_name: str,
+        informed_wallet_pass: str,
```
Comment (TomMcL) [inline vega_sim/scenario/ideal_market_maker_v2/agents.py]: Why is the market maker receiving these?

File: vega_sim/scenario/ideal_market_maker_v2/scenario.py [hunk 5]
```diff
@@ -98,6 +99,7 @@ def __init__(
         self.opening_auction_trade_amount = opening_auction_trade_amount
         self.market_order_trader_base_order_size = market_order_trader_base_order_size
         self.settle_at_end = settle_at_end
+        self.proportion_taken = proportion_taken
```
Comment (TomMcL) [inline vega_sim/scenario/ideal_market_maker_v2/scenario.py:102]: We don't need these if we aren't adding an informed trader to this scenario

Comment (TomMcL) [reply vega_sim/scenario/ideal_market_maker_v2/scenario.py:102]: (I think let's add, just duplicate the setup from the non `_v2` version)

File: vega_sim/scenario/ideal_market_maker/scenario.py [hunk 6]
```diff
@@ -188,14 +192,26 @@ def set_up_background_market(
...
         env = MarketEnvironment(
             base_agents=[
                 market_maker,
                 tradingbot,
                 randomtrader,
                 auctionpass1,
                 auctionpass2,
-                # liquidityprovider,
+                info_trader,
+                liquidityprovider,
```
Comment (TomMcL) [inline vega_sim/scenario/ideal_market_maker/scenario.py:210]: Looks like liquidity provider was commented out here before, is uncommenting deliberate?

**Review by TomMcL — APPROVED** [review]
LGTM

**Pull request merged**